index.html

```diff
@@ -24,14 +24,205 @@ <h1 class="headline1">Hey There.<br>I'm Femi</h1>
                 don’t hestiate to contact me.</p>
             </div>
             <ul class="container">
-                <li><img src="Images/github.png" alt="github"></li>
-                <li><img src="Images/Linkedinicon.png" alt="linkedin"></li>
-                <li><img src="Images/angellist.png" alt="Angel list"></li>
-                <li><img src="Images/twitter.png" alt="Twitter"></li>
-                <li><img src="Images/medium.png" alt="Medium"></li>
+                <li><a href="#"><img src="Images/github.png" alt="github"></a></li>
+                <li><a href="#"><img src="Images/Linkedinicon.png" alt="linkedin"></a></li>
+                <li><a href="#"><img src="Images/angellist.png" alt="Angel list"></a></li>
+                <li><a href="#"><img src="Images/twitter.png" alt="Twitter"></a></li>
+                <li><a href="#"><img src="Images/medium.png" alt="Medium"></a></li>
             </ul>
             <img class="backgroundImage" src="Images/Illustration-header@2x.png" alt="background image">
             <img class="toggleImage" src="Images/Group.png" alt="background image">
         </section>
+        <section class="work">
+            <h3 class="workhead">My Recent Works</h3>
+            <p class="workhead2">My Recent Works</p>
+            <hr>
+            <div class="workflex">
+                <img class="workimage" src="Images/ImgPlaceholder.png" alt="Recent work">
+                <div class="work1">
+                    <p class="work1head">Multi-Post Stories</p>
+                    <p class="work1text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <ul>
+                        <li class="work1p"><p>css</p></li>
+                        <li class="work1p"><p>html</p></li>
+                        <li class="work1p"><p>bootstrap</p></li>
+                        <li class="work1p"><p>Ruby</p></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+                </div>
+                <div class="workgrid">
+                <div class="workgrid1">
+                    <button type="button">See Project</button>
+                </div>
+                <div class="workgrid2">
+                    <p class="workgridp1">Data Dashboard<br>Healthcare</p>
+                    <p class="workgridp2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li><p>html</p></li>
+                        <li><p>bootstrap</p></li>
+                        <li><p>Ruby</p></li>
+                    </ul>
```

Comment:
- [ ] The title for each project card should use a header tag(e.g. `h2`, `h3`, `h4`) instead of `p` as stated in the requirements